cypress/e2e/homepage_spec/UserLogin.cy.ts

```diff
@@ -0,0 +1,45 @@
+import LoginPage from "pageobject/Login/LoginPage";
+const loginPage = new LoginPage();
+
+describe("Authorisation/Authentication", () => {
+  beforeEach(() => {
+    cy.awaitUrl("/", true);
+  });
+
+  it("Try login as admin with correct password", () => {
+    loginPage.loginManuallyAsDistrictAdmin();
+    loginPage.interceptFacilityReq();
+    loginPage.verifyFacilityReq();
+    loginPage.ensureLoggedIn();
+    loginPage.clickSignOutBtn();
+    loginPage.verifyLoginPageUrl();
+  });
+
+  it("Try login as admin with incorrect password", () => {
+    loginPage.interceptLoginReq();
+    loginPage.loginManuallyAsDistrictAdmin(false);
+    loginPage.verifyLoginReq();
+    cy.verifyNotification("No active account found with the given credentials");
+  });
+});
+
+describe("Forgot Password", () => {
```

Comment:
@JavidSumra read the issue properly, it was mentioned to combine all the three test to one single test

Reply:
![IMG_20241111_132942.jpg](https://github.com/user-attachments/assets/7b5c7c2c-8ba9-4f0b-8041-a742e1ea126d)

Reply:
![IMG_20241111_132942.jpg](https://github.com/user-attachments/assets/7b5c7c2c-8ba9-4f0b-8041-a742e1ea126d)